src/lib.rs

```diff
@@ -37,6 +41,7 @@ impl SimpleLogger {
         SimpleLogger {
             default_level: LevelFilter::Trace,
             module_levels: Vec::new(),
+	    timestamps: true
```

Comment:
indent problem?

Reply:
Sorry about that, I guess I have to figure out how to convince the Rust mode to use spaces instead of tabs.

Reply:
Does `cargo fmt` works for you?